src/middlewares/auth.middleware.ts

```diff
@@ -42,6 +43,12 @@ export class AuthMiddleware implements NestMiddleware {
         next();
         return;
       }
+      if (!req.user.email_verified) {
+        return res.status(401).send({
+          success: false,
+          errors: ['Email not verified'],
```

Comment:
Could there be a call to action here?  For example: `Please verify your email address`

Reply:
Sure. Thanks!